distribution/docs/src/main/resources/content/_configuring/configuring-fanout-proxy.adoc

```diff
@@ -7,7 +7,7 @@
 
 == {title}
 
-This scenario describes how to configure ${branding} as a fanout proxy such that only queries and resource retrieval requests are processed and create/update/delete requests are rejected.
+Configure ${branding} as a fanout proxy such that only queries and resource retrieval requests are processed and create/update/delete requests are rejected.
```

Comment:
❓ I'm a little confused by this revision. 
It makes it sound like doing this is a command, "Configure x, so that y".

Reply:
I'll take another pass. I was just trying to get rid of the superfluous "This scenario describes...." because (a) it's less active, and (b) there's no "scenario" described.